extensions/visualizations/visualizations.html

```diff
@@ -6,28 +6,34 @@
   <strong><[options.title]></strong>
   <table class="table">
     <tr>
-      <th><[options.column_headers[0]]></th>
-      <th><[options.column_headers[1]]></th>
+      <th ng-repeat="column_header in options.column_headers">
+        <[column_header]>
+      </th>
     </tr>
-    <tr ng-repeat="row in data">
+    <tr ng-repeat="row in data track by $index">
       <td><[row.answer]></td>
       <td><[row.frequency]></td>
+      <td angular-html-bind="addressedCellContent($index)"></td>
```

Comment:
This shouldn't require angular-html-bind since you're not using HTML. You can do an ng-if on the relevant boolean and show Yes or a tick if that boolean is true.

Reply:
Done.